src/app/core/services/rundown-state.service.ts

```diff
@@ -111,6 +113,15 @@ export class RundownStateService implements OnDestroy {
     rundownSubject.next(activeRundown)
   }
 
+  private rehearsalRundownFromEvent(event: RundownRehearseEvent): void {
```

Comment:
This a method name so we should use the verb `rehearse`

Reply:
Fixed